src/Microsoft.Health.Fhir.Shared.Api/Controllers/ImportController.cs

```diff
@@ -201,7 +201,7 @@ private void ValidateImportRequestConfiguration(ImportRequest importData)
                     throw new RequestNotValidException(string.Format(Resources.UnsupportedResourceType, item.Type));
                 }
 
-                if (item.Url == null)
+                if (item.Url == null || !string.IsNullOrEmpty(item.Url.Query))
```

Comment:
Could you add a test for this scenario in ImportControllerTests?

Reply:
Ok, I will add test in next commit.